DirectRouting/src/main/java/com/communication/quickstart/App.java

```diff
@@ -1,32 +1,76 @@
-		.connectionString(connectionString)
-		.buildAsyncClient();
-
-	System.out.println("Setting trunks");
-	sipRoutingAsyncClient.setTrunksWithResponse(asList(
-		new SipTrunk("sbc.us.contoso.com", 1234),
-		new SipTrunk("sbc.eu.contoso.com", 1234)
-	)).block();
-
-	System.out.println("Setting routes");
-	sipRoutingAsyncClient.setRoutes(asList(
-		new SipTrunkRoute("UsRoute", "^\\+1(\\d{10})$").setTrunks(asList("sbc.us.contoso.com")),
-		new SipTrunkRoute("DefaultRoute", "^\\+\\d+$").setTrunks(asList("sbc.us.contoso.com", "sbc.eu.contoso.com"))
-	)).block();
-
-	System.out.println("Finish");
-  }
+    public static void main(String[] args) {
+        System.out.println("Azure Communication Services - Direct Routing Quickstart");
+
+        String connectionString = "https://<RESOURCE_NAME>.communication.azure.com/;accesskey=<ACCESS_KEY>";
+        SipRoutingAsyncClient sipRoutingAsyncClient = new SipRoutingClientBuilder()
+                .connectionString(connectionString)
+                .buildAsyncClient();
+
+        try {
+            listTrunksAndRoutes(sipRoutingAsyncClient);
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.err.println("Listing failed");
+        }
+
+        /*
+        try {
+            setTrunksAndRoutes(sipRoutingAsyncClient);
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.err.println("Setting failed");
+        }*/
+
+        System.out.println("Finish");
+    }
+
+    private static void listTrunksAndRoutes(SipRoutingAsyncClient client) {
+        System.out.println("Listing trunks");
+        List<SipTrunk> trunks = requireNonNull(client.listTrunks().byPage().blockFirst()).getValue();
+        for (SipTrunk trunk : trunks) {
+            System.out.printf(" %s:%d%n", trunk.getFqdn(), trunk.getSipSignalingPort());
+        }
+        if (trunks.isEmpty()) {
+            System.out.println(" There are no SIP trunks.");
+        }
+
+        System.out.println("Listing routes");
+        List<SipTrunkRoute> routes = requireNonNull(client.listRoutes().byPage().blockFirst()).getValue();
+        for (SipTrunkRoute route : routes) {
+            System.out.printf(" %s: \"%s\" -> %s%n", route.getName(), route.getNumberPattern(), String.join(", ", route.getTrunks()));
+        }
+        if (routes.isEmpty()) {
+            System.out.println(" There are no voice routes.");
+        }
+    }
+
+    // Please provide trunks' FQDNs
+    private static void setTrunksAndRoutes(SipRoutingAsyncClient client) {
+        System.out.println("Setting trunks");
+        client.setTrunksWithResponse(asList(
+                new SipTrunk("<your-sbc-fqdn>", 1234),
+                new SipTrunk("<your-second-sbc-fqdn>", 1234)
+        )).block();
+
+        System.out.println("Setting routes");
+        client.setRoutes(asList(
+                new SipTrunkRoute("UsRoute", "^\\+1(\\d{10})$")
+                        .setTrunks(asList("<your-sbc-fqdn>")),
```

Comment:
Reverted to previous "sbc.us.contoso.com" constants.

In SDK samples review I got comment that we should set up the customer as close to success as possible.
Those constants will not work anyway and user will have to provide own values.
(contoso domain is associated with other resource)
Even if we provide different domain it would work only for one customer at the time due to the resource domain collision.

So the placeholders which indicate the user to replace them seemed as best approach.